build.rb

```diff
@@ -138,75 +140,71 @@
   python_checkup
 ]
 
-KEEP_CURRENT_SUFFIX = "_keep_current"
+LOCALES = ["", "es"]
 
-filenames = {
-  "WINDOWS.md" => WINDOWS,
-  "macOS.md" => MAC_OS,
-  "LINUX.md" => LINUX,
-  "WINDOWS#{KEEP_CURRENT_SUFFIX}.md" => WINDOWS_KC,
-  "macOS#{KEEP_CURRENT_SUFFIX}.md" => MAC_OS_KC,
-  "LINUX#{KEEP_CURRENT_SUFFIX}.md" => LINUX_KC
+FILENAMES = {
+  "WINDOWS" => ["WINDOWS", WINDOWS],
+  "macOS" => ["macOS", MAC_OS],
+  "LINUX" => ["LINUX", LINUX],
+  "WINDOWS_keep_current" => ["WINDOWS", WINDOWS_KC],
+  "macOS_keep_current" => ["macOS", MAC_OS_KC],
+  "LINUX_keep_current" => ["LINUX", LINUX_KC]
 }
 
-DEFAULT_SUBS = {
-  "<CODE_EDITOR>" => "VS Code",
-  "<CODE_EDITOR_CMD>" => "code"
+DELIMITERS = {
+  "WINDOWS" => ["\\$WINDOWS_START\n", "\\$WINDOWS_END\n"],
+  "macOS" => ["\\$MAC_START\n", "\\$MAC_END\n"],
+  "LINUX" => ["\\$LINUX_START\n", "\\$LINUX_END\n"]
 }
 
-subs = {
-  "WINDOWS.md" => DEFAULT_SUBS,
-  "macOS.md" => DEFAULT_SUBS,
-  "LINUX.md" => DEFAULT_SUBS
-}
+def load_partial(partial, locale)
+  match_data = partial.match(/setup\/(?<partial>[0-9a-z_]+)/)
+  partial = match_data[:partial] if match_data
+  partial = File.join(locale, partial) unless locale.empty?
```

Comment:
Does that mean you keep the 🇬🇧 version if the partial is not localized? I think the build should fail so we got an alert on a missing translation

Reply:
@krokrob the function looks for 🇬🇧 content in `content.md` and 🇪🇸 content in `es/content.md`. If some 🇪🇸 content is missing the build will indeed fail when we try to open a file or uri that does not exist.

Reply:
this is why the build gha of this branch fails since the 🇪🇸 branch of the web setup is not merged